server.js

```diff
@@ -0,0 +1,291 @@
+'use strict';
+
+const http = require('http');
+const fs = require('fs');
+const qs = require('querystring');
+
+let authenticationObject = {
+  something: 'imadethisup',
+};
+
+let existingFiles = ['css', '404.html', 'index.html', '.keep'];
+
+let elementNameReceived = '';
+let elementSymbolReceived = '';
+let elementAtomicNumberReceived = 0;
+let elementDescriptionReceived = '';
+let fileName = '';
+
+const server = http.createServer((req, res) => {
+  let reqURL = '/index.html';
+
+  if (req.url !== '/') {
+    if (fs.existsSync(`./public${req.url}`)) {
+      reqURL = req.url;
+    } else {
+      reqURL = '/404.html';
+    }
+  }
+
+  if (req.method === 'GET') {
+    fs.readFile(`./public${reqURL}`, 'UTF-8', function(err, data) {
+      if (err) throw err;
+      res.writeHead(200, { 'Content-Type': 'text/html', 'Content-Length': data.length });
+      res.write(data);
+      res.end();
+    });
+  }
+
+  if (req.method === 'POST' || req.method === 'PUT' || req.method === 'DELETE') {
```

Comment:
This actually occurs at the same time as the post, put, and delete conditional below so they would actually fire at the same time rather than one after the other.

Reply:
The proper way to fix this would be to put your post, put, and delete work into functions and call them from line 54 appropriately.